README.md

```diff
@@ -163,40 +166,16 @@ A JSON object to describe the profilers to be used and their arguments. Below ar
-The sample interval is configurable but advised to keep under 120 seconds as the framestats command returns only data from frames rendered in the past 120 seconds as described [here](https://developer.android.com/training/testing/performance).
-Shorter sample intervals will not cause duplication in the frames gathered as only unique frames are kept.
-The default subject aggregation consists of combining both the frametimes as the delayed frames count in single files for easy further processing.
+| Name (quality)                                                                       | Description                                                                                                                                                                                                                                                                                 |
+|--------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [batterystats](./AndroidRunner/Plugins/batterystats/README_Batterystats.md) (Energy) | Uses the `batterystats` utility and estimates energy consumption via the algorithm proposed in [this article](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7884613&casa_token=oEEnY7XOip8AAAAA:AyRZxwboUh55-n9vmW5NGT62mL_hv85T4wPGWlDQGJ36VpF3bcAV1ufvYBhsYxlB0lIMOYJ_Hc-O&tag=1). |
+| [monsoon](./AndroidRunner/Plugins/monsoon/README_Monsoon.md) (Energy)                | Collects energy consumption via the Monsoon hard-ware profiler and the [Physalia tool](https://github.com/TQRG/physalia).                                                                                                                                                                   |
+| [trepn](./AndroidRunner/Plugins/trepn/README_Trepn.md) (mixed)                       | Collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.                                                                                                                                                                                         |
+| [mem-CPU](./AndroidRunner/Plugins/android/README_Android.md) (Performance)           | Collects memory and CPU usage via the `cpuinfo` and `meminfo` Android utilities.                                                                                                                                                                                                            |
```

Comment:
Add link to [adb dumpsys](https://developer.android.com/studio/command-line/dumpsys) here.

Reply:
Fixed